build/int.cloudbuild.yaml

```diff
@@ -127,6 +127,22 @@ steps:
     - verify lb-http-separate-frontend-and-backend
   name: 'gcr.io/cloud-foundation-cicd/$_DOCKER_IMAGE_DEVELOPER_TOOLS:$_DOCKER_TAG_VERSION_DEVELOPER_TOOLS'
   args: ['/bin/bash', '-c', 'cft test run TestSeparateFrontendAndBackend --stage teardown --verbose']
+  # Internal cross regional http load balancer with cloud-run
+- id: apply internal-lb-http
```

Comment:
GCB run is failing because of 2 test running at the same time. Please include wait on create like it has been done here. https://github.com/terraform-google-modules/terraform-google-lb-http/blob/main/build/int.cloudbuild.yaml#L115-L117